.gitignore

```diff
@@ -1,2 +1,4 @@
 .DS_Store
-.idea/
\ No newline at end of file
+.idea/
+
+node_modules/
```

Comment:
Do we need to ignore this here? I think this is already ignored in the client_vscode files

Reply:
it tries to track it when I don't include it